site-scrapers/Color/index.js

```diff
@@ -1,43 +1,26 @@
-const { site } = require("./config");
 const https = require("https");
+const sites = require("./config");
 
-module.exports = async function GetAvailableAppointments() {
-    console.log(`${site.name} starting.`);
-    const webData = await ScrapeWebsiteData();
-    console.log(`${site.name} done.`);
-    return {
-        ...site,
-        ...webData,
-        timestamp: new Date(),
-    };
-};
+const token = "bcd282a6fe22e6fc47e14be11a35b33fe1bc";
```

Comment:
Did you mean to leave this token hardcoded? Should we be concerned about it expiring?